js/bundle.go

```diff
@@ -334,6 +334,10 @@ func (b *Bundle) instantiate(logger logrus.FieldLogger, rt *goja.Runtime, init *
 	unbindInit()
 	*init.ctxPtr = nil
 
+	if vuID == 0 {
+		init.allowOnlyOpenedFiles()
+	}
```

Comment:
I am probably be missing something, but this seems wrong? :confused: Isn't the whole point that VUs > 0 shouldn't be able to open non-cached files?

Reply:
Yes, the point is that VUs > 0 shouldn't be able to open non-cached files.

This line notifies the FS that VU#0 did the first run, and the list of "known" files list can be frozen.